.gitignore

```diff
@@ -41,6 +41,7 @@ resharper/src/.idea/**/*.iml
 
 # ContentModel.xml is also auto-generated
 resharper/src/.idea/**/contentModel.xml
+resharper/src/.idea/**/other.xml
```

Comment:
Can we stop `other.xml` from being written? @kskrygan 

https://github.com/JetBrains/resharper-unity/blob/master/rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/ui/UnityUIManager.kt#L19